pages/dkp/kommander/1.4/clusters/attach-cluster/finish-attaching-cluster/index.md

```diff
@@ -7,10 +7,17 @@ excerpt: How to apply the kubeconfig file to create the network tunnel to attach
 beta: true
 ---
 
-Though the required kubeconfig file is now generated, you still need to apply it to create the network tunnel and complete the attachment process.
+Though the required file is now generated, you still need to apply it to create the network tunnel and complete the attachment process.
 
-1. Select the **Download Manifest** link to download the kubeconfig file you generated previously.
+1. Select the **Download Manifest** link to download the file you [generated previously][attach-with-restrctions].
 
 1. Copy the `kubectl apply...` command from the user interface and paste into your terminal session, substituting the actual name of the file for the variable. Running this command starts the attachment process, which may take several minutes to complete. The Cluster details page will be displayed automatically when the cluster attachment process completes.
 
 1. (Optional) Select the **Verify Connection to Cluster** button to send a request to Kommander to refresh the connection information. You can use this option to check to see if the connection is complete, though the Cluster Details page displays automatically when the connection is complete.
+
+<p class="message--note"><strong>NOTE: </strong>After the initial connection is made, and your cluster become viewable as attached in the Kommander UI, the attachment and federated addons and platform services will still need to complete.
+This may take several additional minutes.
+If a cluster has limited resources to deploy all the federated platform services, it will fail to stay attached in the Kommander UI.
```

Comment:
The attachment won't fail but the federation of the resources would fail.

Reply:
From my observations, the cluster will fail to stay attached in the UI if it's too small. It may still be attached and viewable via kubectl, but will be in error in the UI. I can adjust the language, though.